freanalysis_aerosol/freanalysis_aerosol/__init__.py

```diff
@@ -11,7 +11,7 @@
 @dataclass
 class Metadata:
     """Helper class that stores the metadata needed by the plugin."""
-    frequency: str = "monthly"
+    frequency: str = "monthly"object
```

Comment:
here and L210- remove `object`

Reply:
Will have to check with @menzel-gfdl on this change